README.md

```diff
@@ -19,7 +19,6 @@ This repository is split into several crates:
 - [unimarkup-core](/core/README.md) ... This crate contains the core reference implementation
 - [unimarkup-inline](/inline/README.md) ... This crate contains the reference compiler for inline elements
 - [unimarkup-render](/render/README.md) ... This crate contains traits needed in **core** and **inline** to render Unimarkup content to supported output formats
```

Comment:
Should we add link and description for commons crate?

Reply:
ah yes. we should 😄